new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/CdcPipelineConfiguration.java

```diff
@@ -74,7 +73,7 @@ public void initialize() {
 
     Optional
             .ofNullable(cdcPipelineJsonProperties)
```

Comment:
This can be expressed much more simply as

if (cdcPipelineReaderJsonProperties != null) {
 ....
}

Reply:
I thought, that is good practice to use optional instead of null checks.
It is not true?